src/dom/query.ts

```diff
@@ -86,12 +88,26 @@ function hitTest(node: Element, clientX: number, clientY: number): boolean {
  * ```
  */
 export
-function scrollIfNeeded(area: HTMLElement, elem: HTMLElement, threshold = 0): void {
+function scrollIfNeeded(area: HTMLElement, elem: HTMLElement): void {
```

Comment:
But it is a specialization, and it's not the mozilla `...IfNeeded` algorithm either, is it?